.stylelintrc.json

```diff
@@ -8,13 +8,13 @@
     "scss/at-rule-no-unknown": true,
     "scss/dollar-variable-default": true,
     "scss/dollar-variable-first-in-block": [true, { "ignore": ["comments", "imports"] }],
-    "scss/dollar-variable-pattern": "^(example|utrecht)-[a-z0-9-]+$",
-    "scss/percent-placeholder-pattern": "^(example|utrecht)-[a-z0-9-]+$",
+    "scss/dollar-variable-pattern": "^(frameless)-[a-z0-9-]+$",
+    "scss/percent-placeholder-pattern": "^(frameless)-[a-z0-9-]+$",
     "scss/operator-no-newline-after": null,
     "scss/at-extend-no-missing-placeholder": null,
-    "custom-property-pattern": "^_?(example|denhaag|utrecht)-[a-z0-9-]+$",
-    "selector-class-pattern": "^(example|denhaag|utrecht)-[a-z0-9_-]+$",
-    "keyframes-name-pattern": "^(example|utrecht)-[a-z0-9-]+$",
+    "custom-property-pattern": "^_?(frameless)-[a-z0-9-]+$",
```

Comment:
Example vervangen door frameless is genoeg. Je wil denhaag en utrecht prefixes nog wel kunnen gebruiken zodat je in custom css eventueel die properties een eigen waarde kan geven.